config/detekt/detekt.yml

```diff
@@ -79,14 +79,15 @@ complexity:
     threshold: 60
   LongParameterList:
     active: true
-    threshold: 6
```

Comment:
Instead of these changes it is acceptable for a conversion task to run the command `gradlew detektBaseline` to ignore these issues